alkira/data_source_alkira_billing_tag.go

```diff
@@ -0,0 +1,45 @@
+package alkira
+
+import (
+	"strconv"
+
+	"github.com/alkiranet/alkira-client-go/alkira"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceAlkiraBillingTag() *schema.Resource {
+	return &schema.Resource{
+		Description: "Use this data source to get information on an existing billing tag.",
+
+		Read: dataSourceAlkiraBillingTagRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Description: "The name of the billing tag.",
+				Type:        schema.TypeString,
+				Required:    true,
+			},
+			"billing_tag_id": {
```

Comment:
Would it be better to use the same attribute name as in `resourceAlkiraBillingTag` with `tag_id` instead of `billing_tag_id`?

Reply:
we do have other "tag" introduced later on, hence the change was made here... actually, id should be sufficient... `data.alkira_billing_tag.id` is the way we will use this, neither billing_tag_id or `tag_id` is necessary...